src/widgets/VaccineChart.js

```diff
@@ -155,40 +265,77 @@ export class VaccineChart extends React.Component {
   }
 }
 
+const chartSize = {
+  width: 1,
+  height: 1.025,
+};
+
 const chartStyles = {
   display: 'flex',
   flexDirection: 'row',
   justifyContent: 'center',
 };
 
+const axisStyles = {
+  fontSize: 15,
+  fontFamily: APP_FONT_FAMILY,
+  fill: '#909192',
+};
+
+const minBoundaryStyles = {
+  stroke: '#70b4f0',
+  opacity: 0.3,
+};
+
+const maxBoundaryStyles = {
+  stroke: '#e95c30',
```

Comment:
this color is SUSSOL_ORANGE, can also import it.

By the way the hazard icon fill was also SUSSOL_ORANGE in the last build that was sent to Sera and also the offset for hazard button was as per this diff: https://github.com/openmsupply/mobile/compare/vaccine-development...vm-chart-rough-styles

Reply:
Fixed by 4a915d3, 473ec13.